docs/shared/packages/cypress/cypress-component-testing.md

```diff
@@ -1,7 +1,7 @@
 # Cypress Component Testing
```

Comment:
There's an issue where when you use the `shell` code fence it doesn't render properly with multiple commands. It also shouldn't, because `shell` is intended to render just one command

![image](https://github.com/nrwl/nx/assets/542458/3cdc3e4d-689e-4734-88a1-801e5b8ba5a5)

So either we use one per command or use the `plaintext` or `plain` (can't remember) to just show the commands. In this particular case we could also use a tab component for react/angular and show the individual commands there. Would be nicer probably

There are two of them which are broken on this page. Maybe we should do a regex search on the docs because there might potentially be more of these

Reply:
fixed the shell component